resources/lrsql/config/prod/default/webserver.edn

```diff
@@ -18,4 +18,5 @@
  :oidc-audience             #env LRSQL_OIDC_AUDIENCE
  :oidc-client-id            #env LRSQL_OIDC_CLIENT_ID
  :oidc-client-template      #or [#env LRSQL_OIDC_CLIENT_TEMPLATE "config/oidc_client.json.template"]
- :oidc-verify-remote-issuer #or [#env LRSQL_OIDC_VERIFY_REMOTE_ISSUER true]}
+ :oidc-verify-remote-issuer #or [#env LRSQL_OIDC_VERIFY_REMOTE_ISSUER true]
+ :oidc-enable-local-admin   #boolean #or [#env LRSQL_OIDC_ENABLE_LOCAL_ADMIN false]}
```

Comment:
Not really related to this PR but shouldn't `:oidc-verify-remote-issuer` also have a `#boolean` tag?

Reply:
It should!

Reply:
https://github.com/yetanalytics/lrsql/pull/217/commits/55befad2a048fb541a42b00ef2efb210cdede65d